crates/orderbook/src/orderbook.rs

```diff
@@ -367,11 +388,15 @@ impl Orderbook {
         let quote_id = quote.as_ref().and_then(|quote| quote.id);
 
         self.database
-            .replace_order(&old_order.metadata.uid, &validated_new_order, quote)
+            .replace_order(&old_order.metadata.uid, &validated_new_order, quote.clone())
             .await
             .map_err(|err| AddOrderError::from_insertion(err, &validated_new_order))?;
-        Metrics::on_order_operation(&old_order, OrderOperation::Cancelled);
-        Metrics::on_order_operation(&validated_new_order, OrderOperation::Created);
+        Metrics::on_order_operation(&old_order, OrderOperation::Cancelled, quote.as_ref());
```

Comment:
The old order was likely created with a different quote which means this could be counted as a different order type here.